packages/x-tailwindcss/demo/src/components/xds-icon.vue

```diff
@@ -0,0 +1,170 @@
+<template>
+  <XdsBaseShowcase
+    #default="{ cssClass, section, copyCssClassesToClipboard, removeClassPrefix }"
+    title="Icon"
+    :sections="sections"
+  >
+    <h3 class="x-text-xxs">{{ removeClassPrefix(cssClass, base) }}</h3>
+    <slot v-bind="{ cssClass, section, copyCssClassesToClipboard }">
+      <div class="x-flex">
+        <CheckIcon @click.native="copyCssClassesToClipboard" :class="cssClass" />
```

Comment:
I am not sure if this is a problem or not, if you wanna copy the classes of the element, if you click in the path, you will not get classes:

![image](https://user-images.githubusercontent.com/6597815/200867452-f18e5f55-7719-449a-a2f3-048b49ca9e2e.png)

Reply:
fixed changing `target` by `currentTarget`